Library/Formula/libmaxminddb.rb

```diff
@@ -1,8 +1,8 @@
 class Libmaxminddb < Formula
   desc "C library for the MaxMind DB file format"
   homepage "https://github.com/maxmind/libmaxminddb"
-  url "https://github.com/maxmind/libmaxminddb/releases/download/1.0.4/libmaxminddb-1.0.4.tar.gz"
-  sha1 "57548d426d43b9b43c77786b08594d48d0c88c62"
+  url "https://github.com/maxmind/libmaxminddb/releases/download/1.1.1/libmaxminddb-1.1.1.tar.gz"
+  sha1 "5ab295438ad914e704d69e3636fc2d951ab97e27"
```

Comment:
We've deprecated the use of `sha1`, could you upgrade this to the `sha256`? Thanks!